modules/ibi-install-ibi-operator.adoc

```diff
@@ -0,0 +1,55 @@
+// Module included in the following assemblies:
+//
+// * edge_computing/ibi-edge-image-based-install.adoc 
+
+:_mod-docs-content-type: PROCEDURE
+[id="ibi-install-ibi-operator_{context}"]
+= Installing the Image Based Install Operator
+
+The Image Based Install (IBI) Operator is part of the image-based deployment workflow for bare-metal hosts preinstalled with {sno}. 
+
+[NOTE]
+====
+The IBI Operator is part of the multicluster engine for Kubernetes operator (MCE) from MCE version 2.7.
+====
+
+.Prerequisites
+
+* You logged in as a user with `cluster-admin` privileges.
+* You deployed a {rh-rhacm-first} hub cluster or you deployed the multicluster engine for Kubernetes operator (MCE).
+* You reviewed the required versions of software components in the section "Software prerequisites for an image-based installation".
+
+.Procedure
+. Edit the `MultiClusterEngine` resource by running the following command:
++
+[source,terminal]
+----
+$ oc edit multiclusterengine
+----
+
+. Set the `enabled` specification to `true` for the `image-based-install-operator` component:
++
+[source,text]
+----
+...
+  - enabled: true
+    name: image-based-install-operator
+...
+----
+
+. Save the file.
```

Comment:
It's not a file.

We can replace the above with a single command:

        oc patch multiclusterengines.multicluster.openshift.io multiclusterengine --type='merge' \
        -p='{"spec": {"overrides": {"components": [{"name": "image-based-install-operator","enabled": true}] }}}'